src/dlgButtonSelect.h

```diff
@@ -0,0 +1,63 @@
+#ifndef MUDLET_DLGBUTTONSELECT_H
+#define MUDLET_DLGBUTTONSELECT_H
+
+/***************************************************************************
+ *   Copyright (C) 2008-2011 by Heiko Koehn - KoehnHeiko@googlemail.com    *
+ *   Copyright (C) 2014 by Ahmed Charles - acharles@outlook.com            *
+ *   Copyright (C) 2016 by Stephen Lyons - slysven@virginmedia.com         *
+ *   Copyright (C) 2017 by Ian Adkins - ieadkins@gmail.com                 *
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ *   This program is distributed in the hope that it will be useful,       *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU General Public License for more details.                          *
+ *                                                                         *
+ *   You should have received a copy of the GNU General Public License     *
+ *   along with this program; if not, write to the                         *
+ *   Free Software Foundation, Inc.,                                       *
+ *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
+ ***************************************************************************/
+
+#include "pre_guard.h"
+#include "ui_button_select.h"
+#include <QGamepad>
+#include "post_guard.h"
+
+class dlgButtonSelect : public QDialog , public Ui::button_select_dlg
+{
+    Q_OBJECT
+
+public:
+
+    dlgButtonSelect(QWidget * parent = 0);
+//    void accept() override;
```

Comment:
:bulb:  You are not `override`-ing the `accept()` (virtual ?) function but, if you do in the future, consider the Qt provided:
> `Q_DECL_OVERRIDE`
>This macro can be used to declare an overriding virtual function. Use of this markup will allow the compiler to generate an error if the overriding virtual function does not in fact override anything.
>It expands to "override" if your compiler supports that C++11 contextual keyword, or to nothing otherwise.
>The macro goes at the end of the function, usually after the const, if any:
>
     // generate error if this doesn't actually override anything:
     virtual void MyWidget::paintEvent(QPaintEvent*) Q_DECL_OVERRIDE;
>
>This function was introduced in Qt 5.0.